src/pal/tests/palsuite/threading/DuplicateHandle/test7/test7.cpp

```diff
@@ -136,14 +158,21 @@ int __cdecl main(int argc, char* argv[])
     }
 
     /* Clean-up thread and Terminate the PAL.*/
+    CloseHandle(hSyncEvent);
     CloseHandle(hThread);
     CloseHandle(hDupThread);
     PAL_Terminate();
     return PASS;
 }
 
-/*Thread testing function, only return '0'*/
+/*Thread testing function*/
 DWORD PALAPI CreateTestThread(LPVOID lpParam)
 {
+    HANDLE hSyncEvent = (HANDLE)lpParam;
+
+    /* Wait until the main thread signals that this helper thread shoudl shut down */
```

Comment:
"shoudl" => "should"

Reply:
Thanks for spotting it, I'll amend the commit and merge it in.